examples/lm1b/train.py

```diff
@@ -450,34 +453,34 @@ def encode_strings(strs, max_len):
   initial_variables = jax.jit(m.init)(init_rng,
                                       jnp.ones(input_shape, jnp.float32))
 
-  # apply an optimizer to this tree
-  optimizer_def = optim.Adam(
-      config.learning_rate,
-      beta1=0.9,
-      beta2=0.98,
-      eps=1e-9,
-      weight_decay=config.weight_decay)
-  optimizer = optimizer_def.create(initial_variables["params"])
+  learning_rate_fn = create_learning_rate_scheduler(
+      base_learning_rate=config.learning_rate, warmup_steps=config.warmup_steps)
 
-  # We access model params only from optimizer below via optimizer.target.
+  optimizer = optax.adamw(
+      learning_rate_fn, b1=0.9, b2=0.98, eps=1e-9,
+      weight_decay=config.weight_decay
+      )
+  opt_state = train_state.TrainState.create(
```

Comment:
We usually call this simply `state`, and not `opt_state`.

Reply:
Done.